pkgs/applications/misc/dupeguru/default.nix

```diff
@@ -0,0 +1,62 @@
+{stdenv, python, gettext, qt5, fetchFromGitHub}:
+
+stdenv.mkDerivation rec {
+  pname = "dupeguru";
+  version = "4.0.4";
+
+  src = fetchFromGitHub {
+    owner = "arsenetar";
+    repo = "dupeguru";
+    rev = "${version}";
+    sha256 = "0ma4f1c6vmpz8gi4sdy43x1ik7wh42wayvk1iq520d3i714kfcpy";
+    fetchSubmodules = true;
+  };
+
+  build_python = python.withPackages (ps: [ps.setuptools]);
+
+  nativeBuildInputs = [ 
+    gettext 
+    build_python
+    qt5.wrapQtAppsHook
+  ];
+
+  app_python = python.withPackages (ps: with ps; [
+      pyqt5
+      send2trash
+      sphinx
+      polib
+      hsaudiotag3k
+  ]);
+
+  buildInputs = [
+    app_python
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+    make i18n
+    make modules
+    make qt/dg_rc.py
+    make pyc
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    make PREFIX=$out install --assume-old=all --assume-old=pyc
+    chmod +x $out/share/dupeguru/run.py
+    runHook postInstall
+  '';
+
+  doCheck = false;
+
+  postFixup = ''
+    wrapQtApp $out/bin/dupeguru
```

Comment:
There should not be a need to call this manually, unless you disable it.

Reply:
Though I actually recommend the following to avoid double wrapping.

```
  dontWrapQtApps = true;

  preFixup = ''
    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
  '';
```